wiki/addnodes.md

```diff
@@ -18,12 +18,13 @@ See the [config file](config-file "wikilink") page about how to change your conf
 You can also add the node with the command: `addnode <node> <add|remove|onetry>` 
 in the debug console.
 
+You shouldn't need to add them manually as a few are hardcoded into the wallet.
```

Comment:
This is not always the case. It can be helpful to add more sometimes. For instance, users who ran old versions for too long may be banned by those default addnodes.